travis-ci.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Don't fail silently when a step doesn't succeed
+set -e
+
+# if [ -z "$TRAVIS" ]; then
```

Comment:
Do we want this uncommented or removed?

Reply:
I'll remove, I find it handy to kick it all off locally.